### ISSUE TITLE
prestashop generated track & trace issues

### ISSUE THREAD

**Issue by goldleewtje** [wienkit/prestashop-bolplaza#6]
when my boss gets a order in thru bol.com and he proccesses the order the track & trace code doesnt get send to bol.com any idea how to fix that issue? i have 0,0 experience with presta

**Comment by coMegaWebMaster** [comment]
Sometimes this happens when an module that is creating the track & trace and there is a delay between the two.
A quick fix is to open that order and at shipping click the "Edit" next to the Tracking number (the tracking number should be already be there) and then just click on "Update", now it is being synced with Bol.
Hopefully there will be a better solution for this later, but this is what I discovered and I hopes it helps.

**Comment by goldleewtje** [comment]
apparently the order doesnt get processed in the cronjob thats why the track and trace code doesnt get send automaticly to the customer. but is that a issue with the module or an issue somewhere else?

**Comment by coMegaWebMaster** [comment]
The tracking number update is immediately, that is not a cronjob. If my last comment did not work then there is something wrong with:
- the tracking number or
- how it is setup.

**Comment by goldleewtje** [comment]
but according to my boss. when a product gets ordered on bol.com they have a cronjob that sets it in the system. but that cronjob doesnt work like it supposed to.

**Comment by coMegaWebMaster** [comment]
The cronjob get the orders, those orders will be in PrestaShop and if the setup is done correctly (https://werkaandewebshop-bolplaza.readthedocs.io/nl/latest/) then they will be updated when the tracking number is added on that order in PS, this happens as said, immediately.

- What PS version are you using?
- How long are you using the Bol module?
- Is this the first time you have this issue?

**Comment by goldleewtje** [comment]
the presta version is: 1.7.5.1
they have been using the module for a while now i think and they never had this issue before.

**Comment by coMegaWebMaster** [comment]
And is it now all the time like this, none of the orders are being put on send in Bol when the tracking number is made?

**Comment by goldleewtje** [comment]
when they get an order thru bol they dont get put in the prestashop bol.com orders tab. as if the cronjob doesnt sync the prestashop and bol.com

**Comment by coMegaWebMaster** [comment]
OK, but that is another issue then you stared with. I think something is wrong with your API setting. Did you check if it is correct in Bol and in the Module?
https://werkaandewebshop-bolplaza.readthedocs.io/nl/latest/configuratie/module/
https://werkaandewebshop-bolplaza.readthedocs.io/nl/latest/configuratie/cron/
You can always make an new API and also check if it is in (https://www.bol.com/sdd/settings.html#/services/api) if the API keys for production is being used, maybe someone made a change, always check the basics first if everything is correct.
Hopefully there is just an setting wrong, otherwise @wienkit has to help.

**Comment by goldleewtje** [comment]
Beste @mwienk ,

De api staat goed aangezien de bestellingen wel worden ingeschoten als bestelling maar niet zichtbaar zijn in het tabje bol orders, hierdoor wordt er geen track en trace teruggekoppeld naar bol.com

**Comment by coMegaWebMaster** [comment]
Question, what module are you using to make the tracking label, is it the MyParcel module?

**Comment by goldleewtje** [comment]
yeah they are using myparcel everything broke after they upgraded prestashop

**Comment by coMegaWebMaster** [comment]
OK, then I can confirm that issue, but it is not an issue in this Bol module, there will be a fix in the MyParcel soon. As said in my first comment, that quick fix should work. What will work, strangely, is the "bulkacties" and then "Exporteren en printen", select the orders and then choose that option and then it will work, as said this issue will be fixed soon, in the MyParcal module:
https://github.com/myparcelnl/prestashop/releases
So wait for the new version and hopefully you can work with the way how I describe here in the meantime.

**Comment by goldleewtje** [comment]
what does myparcel have to do with the cronjob of the bolplaza module?

**Comment by goldleewtje** [comment]
the orders dont get put in the prestashop. bolplaza doesnt get synced with the prestashop so the orders dont appear in the bol.com orders.

**Comment by coMegaWebMaster** [comment]
That @mwienk has to have a look at it, I can only confirm what I see in other webshops and I cannot simulate your issue, just the MyParcel I can confirm. As said, try to get a new API code and check the cronjob, or try the "Synchroniseer bestellingen" in the "Bestellingen / Bol.com orders" top.
